docs/docs/validators/common.md

```diff
@@ -61,3 +60,36 @@ The message function is given the following arguments:
 The return value must be a `string`. If nothing is returned (`undefined`), defaults to the default error message of the specified type.
 
 Within this function, the context is set to that of the current validator. This gives you access to the model, defaultMessages, options and more.
+
+## createErrorMessage
+This function is used by all pre-defined validators to build an error message that is present in `validators/message`.
+
+The method signature is as follows:
+`createErrorMessage(type, options, value, ...formats) {}`
+
+* `type` (**String**): The error message type
+* `options` (**Object**): The validator options that were defined in the model
+* `value`: The current value being evaluated
+* `formats` (**...Array**): String formatters (%@)
+
+If we extended our default messages to include `uniqueUsername: '"%@" already exists'`, we can use this method to generate our error message.
```

Comment:
That's just currently in place until we include the i18n solution (#9)  which will replace the string.fmt approach we currently have.